From kamalesh@linux.vnet.ibm.com  Fri Nov 21 11:03:21 2008
From: Kamalesh Babulal <kamalesh@linux.vnet.ibm.com>
Date: Thu, 20 Nov 2008 22:39:27 +0530
Subject: Staging: asus_oled: fix build dependancy
Cc: gregkh@suse.de, <sjakub@gmail.com>
Message-ID: <20081120170927.GB4821@linux.vnet.ibm.com>

From: Kamalesh Babulal <kamalesh@linux.vnet.ibm.com>

asus_oled depends on the CONFIG_USB_SUPPORT, I have only build tested
the patch.

Signed-off-by: Kamalesh Babulal <kamalesh@linux.vnet.ibm.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/asus_oled/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/asus_oled/Kconfig
+++ b/drivers/staging/asus_oled/Kconfig
@@ -1,5 +1,6 @@
 config ASUS_OLED
 	tristate "Asus OLED driver"
+	depends on USB
 	default N
 	---help---
 	  Enable support for the OLED display present in some Asus laptops.
